client-app/core/utilities/common/index.ts

```diff
@@ -120,3 +120,22 @@ export const getLinkAttr = (link?: RouteLocationRaw): LinkAttrType => {
   }
   return {};
 };
+
+export function debounceByParamsWithKeys(func: (...args: any[]) => any, delay: number, keyIndices: number[]) {
```

Comment:
https://v10-9-0.vueuse.org/shared/useDebounceFn/#usedebouncefn

Reply:
The key that it debounce functions with different parameters separately, as far as I'm concerned useDebounceFn doesn't pay any attention on parameters. Anyway this PR is going to be closed since we come up with different approach.